### PR TITLE
Incorporate minor feedback changes 

### PR DESCRIPTION
**Pull request by catlu11**
Incorporated some small feedback changes from last PR - merging separately now before creating a new feature branch!
- Renamed ItineraryPreferences -> WaypointPreferences
- Made ISO 8601 method naming more consistent
- Added a TimeInHrMin struct

### CODE REVIEW

File: Trippy/Utils/DateUtils.h [hunk 1]
```diff
@@ -9,11 +9,17 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+typedef struct TimeInHrMin {
+    int hours;
+    int minutes;
+} TimeInHrMin;
```
Comment (ntourte) [inline Trippy/Utils/DateUtils.h:15]: 👍

File: Trippy/Utils/DateUtils.h [hunk 2]
```diff
@@ -9,11 +9,17 @@
...
+
 @interface DateUtils : NSObject
-+ (NSString *)formatDateAsISO8601:(NSDate *)date;
-+ (NSDate *)isoStringToDate:(NSString *)isoString;
++ (NSString *)formatDateAsIso8601:(NSDate *)date;
++ (NSDate *)iso8601StringToDate:(NSString *)isoString;
 + (int)aheadSecondsFrom1970:(NSDate *)date aheadBy:(int)aheadBy;
-+ (NSArray *)secondsToHourMin:(int)seconds;
++ (TimeInHrMin)secondsToHourMin:(int)seconds;
++ (NSNumber *) hourMinToSeconds:(TimeInHrMin)time;
```
Comment (ntourte) [inline Trippy/Utils/DateUtils.h:22]: nit: consider just an `int` return here. `NSNumber` feels heavy (even if you're going to `NSNumber`-ify it later; can quickly do that with a `@(intVariable)`. That way it also has symmetry with your other fn above, which takes in an `int`.

File: Trippy/Utils/DateUtils.m [hunk 3]
```diff
@@ -28,10 +28,15 @@ + (int)aheadSecondsFrom1970:(NSDate *)date aheadBy:(int)aheadBy {
...
++ (struct TimeInHrMin)secondsToHourMin:(int)seconds {
     int hours = seconds / 3600;
     int minutes = (seconds - (hours * 3600)) / 60;
-    return @[[NSNumber numberWithInt:hours], [NSNumber numberWithInt:minutes]];
+    TimeInHrMin time = {.hours = hours, .minutes = minutes};
+    return time;
+}
+
++ (NSNumber *) hourMinToSeconds:(TimeInHrMin)time {
+    return [[NSNumber alloc] initWithInt:time.hours * 3600 + time.minutes * 60];
```
Comment (ntourte) [inline Trippy/Utils/DateUtils.m:39]: PEMDAS, yes; but a good habit (for readability too) to wrap your operation pairs with parentheses.

File: Trippy/View Controllers/Itinerary Tab/PreferencesViewController.m [hunk 4]
```diff
@@ -82,12 +82,13 @@ - (IBAction)tapView:(id)sender {
 - (IBAction)tapUpdate:(id)sender {
     NSDate *etaStart = self.prefEtaSwitch.isOn ? self.etaStartPicker.date : [NSNull null];
     NSDate *etaEnd = self.prefEtaSwitch.isOn ? self.etaEndPicker.date : [NSNull null];
-    NSNumber *stayTime = self.estStaySwitch ? [self stayTimeToSeconds] : @0;
+    TimeInHrMin fieldTime = {.hours = [self.stayHrField.text intValue], .minutes=[self.stayMinField.text intValue]};
```
Comment (ntourte) [inline Trippy/View Controllers/Itinerary Tab/PreferencesViewController.m:85]: Not a big fan of getting values from text fields... can you pull from the source instead? You might even already have it in seconds.

**Review by ntourte — APPROVED** [review]
Just a few things. Nice work!

File: Trippy/View Controllers/Itinerary Tab/PreferencesViewController.m (hunk 4 above)
Comment (catlu11) [reply Trippy/View Controllers/Itinerary Tab/PreferencesViewController.m:85]: I'm getting an input value from the user, so I'm not sure what you mean by source?

Comment (ntourte) [reply Trippy/View Controllers/Itinerary Tab/PreferencesViewController.m:85]: AH! Ok, no worries then 😄 disregard!

**Pull request merged**